tasks/deploy.js

```diff
@@ -0,0 +1,249 @@
+        nbUnits: toBN("100000"),
+        tokenAddr: ETH_TOKEN,
+        erc20TokenName: process.env.ERC20_TOKEN_NAME,
+        erc20TokenSymbol: process.env.ERC20_TOKEN_SYMBOL,
+        erc20TokenDecimals: process.env.ERC20_TOKEN_DECIMALS,
+        maxChunks: toBN("100000"),
+        votingPeriod: parseInt(process.env.VOTING_PERIOD_SECONDS),
+        gracePeriod: parseInt(process.env.GRACE_PERIOD_SECONDS),
+        offchainVoting: true,
+        chainId: getNetworkDetails(network).chainId,
+        deployTestTokens: false,
+        finalize: false,
+        maxExternalTokens: 100,
+        couponCreatorAddress: process.env.COUPON_CREATOR_ADDR,
+        daoName: process.env.DAO_NAME,
+        owner: process.env.DAO_OWNER_ADDR,
+        offchainAdmin: process.env.OFFCHAIN_ADMIN_ADDR,
+    });
+};
+  
+const deployGanacheDao = async (
+    deployFunction,
+    network,
+    accounts,
+    hardHatImports
+    ) => {
+    if (!process.env.DAO_NAME)
+        throw Error("Missing env var: DAO_NAME");
+    if (!process.env.ERC20_TOKEN_NAME)
+        throw Error("Missing env var: ERC20_TOKEN_NAME");
+    if (!process.env.ERC20_TOKEN_SYMBOL)
+        throw Error("Missing env var: ERC20_TOKEN_SYMBOL");
+    if (!process.env.ERC20_TOKEN_DECIMALS)
+        throw Error("Missing env var: ERC20_TOKEN_DECIMALS");
+    return await deployDao({
+        ...hardHatImports,
+        deployFunction,
+        unitPrice: toBN(toWei("100", "finney")).toString(),
+        nbUnits: toBN("100000").toString(),
+        tokenAddr: ETH_TOKEN,
+        erc20TokenName: process.env.ERC20_TOKEN_NAME,
+        erc20TokenSymbol: process.env.ERC20_TOKEN_SYMBOL,
+        erc20TokenDecimals: process.env.ERC20_TOKEN_DECIMALS,
+        maxChunks: toBN("100000").toString(),
+        votingPeriod: 120, // 120 secs = 2 mins
+        gracePeriod: 60, // 60 secs = 1 min
+        offchainVoting: true,
+        chainId: getNetworkDetails(network).chainId,
+        deployTestTokens: true,
+        finalize: false,
+        maxExternalTokens: 100,
+        couponCreatorAddress: process.env.COUPON_CREATOR_ADDR
+            ? process.env.COUPON_CREATOR_ADDR
+            : accounts[0].address,
+        daoName: process.env.DAO_NAME,
+        owner: accounts[0].address,
+        offchainAdmin: "0xedC10CFA90A135C41538325DD57FDB4c7b88faf7",
+    });
+};
+  
+const deployTestDao = async (
+    deployFunction,
+    network,
+    accounts,
+    hardHatImports
+    ) => {
+    if (!process.env.DAO_NAME) throw Error("Missing env var: DAO_NAME");
+    if (!process.env.ERC20_TOKEN_NAME)
+        throw Error("Missing env var: ERC20_TOKEN_NAME");
+    if (!process.env.ERC20_TOKEN_SYMBOL)
+        throw Error("Missing env var: ERC20_TOKEN_SYMBOL");
+    if (!process.env.ERC20_TOKEN_DECIMALS)
+        throw Error("Missing env var: ERC20_TOKEN_DECIMALS");
+  
+    return await deployDao({
+        ...hardHatImports,
+        deployFunction,
+        unitPrice: unitPrice,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/openlawteam/tribute-contracts/pullRequest?prid=8819146)
```suggestion
    unitPrice,
```